contracts/Staking.sol

```diff
@@ -0,0 +1,45 @@
+pragma solidity 0.6.2;
+
+import "./vendor/SafeMath.sol";
```

Comment:
I think we can import the OZ library directly here no need to have our own copy as they already support `0.6`. The same applies to the Storage contract.

Reply:
Yeah, this is already done in `master` for the `StorageManager` contract.

Reply:
Merged master and removed line 3.